GViewCore/src/View/DissasmViewer/Config.cpp

```diff
@@ -38,30 +41,19 @@ void Config::Initialize()
 
     bool foundSettings = false;
     auto ini           = AppCUI::Application::GetAppSettings();
-    if (ini)
-    {
+    if (ini) {
         auto sect = ini->GetSection("DissasmView");
-        if (sect.Exists())
-        {
-            this->Keys.AddNewType              = sect.GetValue("AddNewType").ToKey(Key::F6);
-            this->Keys.ShowFileContentKey      = sect.GetValue("ShowFileContentKey").ToKey(Key::F9);
-            this->Keys.ExportAsmToFile         = sect.GetValue("AsmExportToFile").ToKey(Key::F8);
-            this->Keys.JumpBack                = sect.GetValue("JumpBack").ToKey(Key::Ctrl | Key::Q);
-            this->Keys.JumpForward             = sect.GetValue("JumpForward").ToKey(Key::Ctrl | Key::E);
-            this->Keys.DissasmGotoEntrypoint   = sect.GetValue("Entrypoint").ToKey(Key::F2);
+        if (sect.Exists()) {
```

Comment:
ce se intampla daca unele optiuni noi nu exista?

Reply:
If some keys are missing, they will be initialized with the default value.
Those lines that you have selected have just moved inside Config.hpp. Now all keys are created with their default key combination, and they are updated if they are found inside their ini category.